testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViews.java

```diff
@@ -397,6 +398,36 @@ public void testFromUtcTimestampInvalidTimeZone()
         onTrino().executeQuery("DROP TABLE test_from_utc_timestamp_invalid_time_zone_source");
     }
 
+    @Test(groups = HIVE_VIEWS)
+    public void testNestedFieldWithReservedKeyNames()
+    {
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_nested_field_with_reserved_key_names_source");
+        onHive().executeQuery("CREATE TABLE test_nested_field_with_reserved_key_names_source (nested_field struct<`from`:BIGINT>)");
```

Comment:
Could you add more fields? 
* Reserved keyword with uppercase
* Non reserved keyword, but requires double-quotes in Trino. e.g. `do.t`

Reply:
sounds good, working on it.